R/utilities.R

```diff
@@ -84,16 +84,6 @@ clist <- function(l) {
   paste(paste(names(l), l, sep = " = ", collapse = ", "), sep = "")
 }
 
-# Return unique columns
-# This is used for figuring out which columns are constant within a group
-#
-# @keyword internal
-uniquecols <- function(df) {
-  df <- df[1, sapply(df, is_unique), drop = FALSE]
-  rownames(df) <- seq_len(nrow(df))
-  df
-}
-
```

Comment:
What is the purpose of moving this function?

Reply:
This is a helper is only used in `stat_summary()`, so it feels more natural to be defined there rather than in the 'global' utilities. The only thing moving this does, is preventing me (possibly others too) from wondering 'where is this function used again' every time I search something in the global utilities.

Reply:
fair enough. While you touch it can you make it so it turns off row names rather than set it to numbers (using `.set_row_names(now(df)`)

Reply:
Fixed in 13b8d36